src/material/chips/chip-row.ts

```diff
@@ -212,4 +214,7 @@ export class MatChipRow extends MatChip implements AfterViewInit {
   private _getEditInput(): MatChipEditInput {
     return this.contentEditInput || this.defaultEditInput!;
   }
+
+  /** Id of a span that contains this chip's aria description. @docs-private */
+  _ariaDescriptionId = `chip-row-${id++}-aria-description`;
```

Comment:
nit: it looks like we're repeating this in a couple of places. Maybe we can move it into the base `MatChip` class?